src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -24,7 +24,7 @@ const ProfilePicture = ({
   return (
     <View style={styles.wrapImageProfile}>
       <TouchableNativeFeedback onPress={onImageContainerClick}>
-        <CircleGradient fill={karmaScore} size={normalize(100)} width={normalize(2.2)}>
+        <CircleGradient fill={karmaScore} size={normalize(100)} width={normalize(6)}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The width of the `CircleGradient` component has been increased from `normalize(2.2)` to `normalize(6)`. This change will make the circular border around the profile picture thicker. Ensure this is the intended visual effect and that it doesn't obscure important parts of the profile image or clash with other UI elements.

```diff
- <CircleGradient fill={karmaScore} size={normalize(100)} width={normalize(2.2)}>
+ <CircleGradient fill={karmaScore} size={normalize(100)} width={normalize(6)}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->